tools/templateTests/Template_1.2019-04-01.json

```diff
@@ -788,21 +788,21 @@
       }
     },
     {
-      "name": "[parameters('redisName')]",
       "type": "Microsoft.Cache/Redis",
-      "location": "[parameters('redisLocation')]",
-      "apiVersion": "2014-04-01-preview",
-      "dependsOn": [],
-      "tags": {
-        "displayName": "redis"
```

Comment:
Is it intended to remove the old 'redis' displayName?

Reply:
The `2014-04-01-preview` apiVersion is no longer declared in the [specs repo](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/redis/resource-manager), so the generator removed it, which broke this test. Rather than trying to fix it manually, I simply copy-pasted the example from tools/templateTests/Template_2.2015-01-01.json which is referencing the newer API version 2016-04-01.